system-tests/config-files/local_file_writer_config.ini

```
@@ -0,0 +1,5 @@
+hdf-output-prefix="./output-files/"
+command-uri="//localhost:9092/TEST_writerCommand"
```

```suggestion
command-uri="localhost:9092/TEST_writerCommand"
```
The ridiculousness of using two forward slashes without specifying a protocol is ridiculous and used *nowhere else*.